bootstrapvz/common/tasks/ssh.py

```diff
@@ -109,4 +110,8 @@ def run(cls, info):
         public = [path + '.pub' for path in private]
 
         from ..tools import log_check_call
-        log_check_call(['shred', '--remove'] + private + public)
+        for ssh_host_key in private + public:
+            if os.path.isfile(ssh_host_key):
```

Comment:
Yes, your piece of code is more pythonic :-) I just changed it and rebased last commit.